### PR TITLE
Render the payable text if the resource needs payment

### PR DESCRIPTION
**Pull request by urshala**
If the resource is not free i.e. needs payment but the payment
amounts are missing, render 'Payable' text.

This is a temporary fix for now. All the prices for the resource
will come from a pricelist later.

### CODE REVIEW

File: app/i18n/messages/fi.json [hunk 1]
```diff
@@ -354,6 +354,7 @@
   "ResourceHeader.purpose": "Käyttötarkoitus",
   "ResourceHeader.capacity": "Kapasiteetti",
   "ResourceHeader.maxTime": "Enimmäisvarausaika",
+  "ResourceHeader.payable": "Maksullinen",
```
Comment (wiltso) [inline app/i18n/messages/fi.json:357]:
```suggestion
  "ResourceHeader.payable": "Maksettava",
```

**Review by wiltso — COMMENTED** [review]
Why is this being changed if you it still needs the price list to be implemented?

File: app/i18n/messages/fi.json (hunk 1 above)
Comment (urshala) [reply app/i18n/messages/fi.json:357]: This is the customer's request for now. We are still using the prices field but those can be empty even if the prices is not free of charge. (In that case customer wants to render 'Maksullinen` ).  The whole prices related stuff is changing soon. We are working on it at the moment, so both backend and frontend will change soon.

**Review by wiltso — APPROVED** [review]
Looks good to me

**Pull request merged**